suite-native/firmware/src/components/FirmwareInstallationScreenContent.tsx

```diff
@@ -87,14 +89,16 @@ export const FirmwareInstallationScreenContent = ({
     const openLink = useOpenLink();
 
     useEffect(() => {
+        if (!isTemporaryRememeberAllowed) return;
+
         // This will prevent device from being forgotten after firmware update, so discovery will not run again
-        dispatch(setDeviceForceRememberedThunk({ forceRemember: true }));
```

Comment:
I think we can remove the Thunk if its not used anywhere `setDeviceForceRememberedThunk`

Reply:
it was renamed to `setTemporaryRememberedDeviceThunk`

Reply:
sorry my bad, the search took me to `libDev` source maps files